app/src/main/res/layout/activity_detailsmovie.xml

```diff
@@ -4,188 +4,220 @@
+                />
 
             <TextView
-                android:id="@+id/longMovieTextView"
+                android:id="@+id/text_run_time"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:layout_below="@id/timeMovieTextView"
+                android:layout_alignBottom="@+id/image_clock"
                 android:layout_marginLeft="@dimen/dp_5"
                 android:layout_marginStart="@dimen/dp_5"
-                android:layout_marginTop="@dimen/dp_8"
-                android:layout_toEndOf="@id/clockImageView"
-                android:layout_toRightOf="@id/clockImageView"
+                android:layout_toEndOf="@id/image_clock"
+                android:layout_toRightOf="@id/image_clock"
                 android:contentDescription="@string/appbar_scrolling_view_behavior"
                 android:text="@string/movielong"
-                android:textStyle="bold" />
+                android:textColor="@color/colorDescription"
+                android:textSize="@dimen/sp_18"
+                />
 
             <ImageView
-                android:id="@+id/kindImageView"
+                android:id="@+id/image_kind_movie"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:layout_below="@id/clockImageView"
+                android:layout_below="@id/image_clock"
                 android:layout_marginLeft="@dimen/dp_12"
                 android:layout_marginStart="@dimen/dp_12"
-                android:layout_marginTop="@dimen/dp_5"
-                android:layout_toEndOf="@id/movieImageView"
-                android:layout_toRightOf="@id/movieImageView"
+                android:layout_marginTop="@dimen/dp_10"
+                android:layout_toEndOf="@id/image_poster"
+                android:layout_toRightOf="@id/image_poster"
                 android:contentDescription="@string/appbar_scrolling_view_behavior"
-                android:src="@drawable/ic_assignment" />
+                android:src="@drawable/ic_assignment"
+                />
 
             <TextView
-                android:id="@+id/kindMovieTextView"
+                android:id="@+id/text_kind_movie"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:layout_below="@id/longMovieTextView"
+                android:layout_alignTop="@+id/image_kind_movie"
+                android:layout_below="@id/text_run_time"
                 android:layout_marginLeft="@dimen/dp_5"
                 android:layout_marginStart="@dimen/dp_5"
-                android:layout_marginTop="@dimen/dp_10"
-                android:layout_toEndOf="@id/kindImageView"
-                android:layout_toRightOf="@id/kindImageView"
+                android:paddingRight="@dimen/dp_10"
+                android:paddingEnd="@dimen/dp_10"
```

Comment:
WARNING: [RtlSymmetry] When you define `paddingEnd` you should probably also define `paddingStart` for right-to-left symmetry
 If you specify padding or margin on the left side of a layout, you should probably also specify padding on the right side (and vice versa) for right-to-left layout symmetry.